Description: populate ejabberd.service
 Turn the service template into an actual service file for systemd
Author: Mathias Ertl <mati@jabber.at>

Index: ejabberd/ejabberd.service.template
===================================================================
--- ejabberd.orig/ejabberd.service.template
+++ ejabberd/ejabberd.service.template
@@ -7,11 +7,13 @@ User=ejabberd
 Group=ejabberd
 LimitNOFILE=16000
 RestartSec=5
-ExecStart=/bin/sh @ctlscriptpath@/ejabberdctl start
-ExecStop=@ctlscriptpath@/ejabberdctl stop
-ExecReload=@ctlscriptpath@/ejabberdctl reload_config
-Type=oneshot
-RemainAfterExit=yes
+ExecStart=/bin/sh /usr/sbin/ejabberdctl start
+ExecStop=/usr/sbin/ejabberdctl stop
+ExecReload=/usr/sbin/ejabberdctl reload_config
+Type=forking
+Restart=on-failure
+PIDFile=/run/ejabberd/ejabberd.pid
 
 [Install]
 WantedBy=multi-user.target
+Alias=ejabberd.service
